mappings/net/minecraft/item/UnsizedItemStackSet.mapping

```diff
@@ -0,0 +1,10 @@
+CLASS net/minecraft/unmapped/C_voakmxnc net/minecraft/item/UnsizedItemStackSet
+	FIELD f_vjorabmg HASH_STRATEGY Lit/unimi/dsi/fastutil/Hash$Strategy;
+	METHOD m_llcvfzzg hashCode (Lnet/minecraft/unmapped/C_sddaxwyk;)I
+	METHOD m_nnsofwir get ()Ljava/util/Set;
+	CLASS C_sansacgo
+		METHOD equals (Ljava/lang/Object;Ljava/lang/Object;)Z
+			ARG 1 stack
```

Comment:
```suggestion
			ARG 1 stack1
```
If there's a `stack2` it seems like the first should be `stack1`?

Reply:
that's carried over from the old class, I'll look and see if I can name them something more descriptive

Reply:
oh wait never mind that's an equals method